aiodnsprox/coap.py

```diff
@@ -166,10 +147,21 @@ async def create_server(self, loop, *args, local_addr=None, **kwargs):
         )
         site.add_resource(['dns-query'], self.DNSQueryResource(self))
 
-        aiocoap.transports.tinydtls_server.securitystore = \
-            self._CredentialStore()
+        try:
+            client_identity = Config()['dtls_credentials']['client_identity']
+            psk = Config()['dtls_credentials']['psk']
+        except KeyError as exc:
+            raise RuntimeError(f"DTLS credential option {exc} not found")
```

Comment:
```suggestion
            raise RuntimeError(f"DTLS credential option {exc} not found") \
                from exc
```

Reply:
That shouldn't be necessary any more since at least Python 3.0 (when raising from an except branch, the current exception is always the cause unless silenced with `from None`) -- is that really what pylint is complaining about here? Checking why they insist...

Reply:
Python docs: "works implicitly if an exception is raised inside an exception handler or a finally clause: the previous exception is then attached as the new exception’s __context__ attribute:".

OK, found it in https://stefan.sofa-rockers.org/2020/10/28/raise-from/ -- I think that pylint is indeed overzealous, but as that's how you're using it, *shrug*, fixing and squashing right away.

Reply:
Thanks! Wouldn't have minded if, with your research attached, you would have suppressed that warning ^^.